pkg/proxy/iptables/proxier.go

```diff
@@ -333,6 +334,42 @@ func NewProxier(ipt utiliptables.Interface,
 	return proxier, nil
 }
 
+// NewDualStackProxier creates a MetaProxier instance, with IPv4 and IPv6 proxies.
+func NewDualStackProxier(
+	ipt [2]utiliptables.Interface,
+	sysctl utilsysctl.Interface,
+	exec utilexec.Interface,
+	syncPeriod time.Duration,
+	minSyncPeriod time.Duration,
+	masqueradeAll bool,
+	masqueradeBit int,
+	clusterCIDR [2]string,
+	hostname string,
+	nodeIP [2]net.IP,
+	recorder record.EventRecorder,
+	healthzServer healthcheck.ProxierHealthUpdater,
+	nodePortAddresses []string,
+) (proxy.Provider, error) {
+	// Create an ipv4 instance of the single-stack proxier
+	ipv4Proxier, err := NewProxier(ipt[0], sysctl,
```

Comment:
ipt[0] and ipt[1] can have `ipv4, ipv6` or `ipv6,ipv4` as per the code in https://github.com/kubernetes/kubernetes/pull/82462/files#diff-934760ea1a4e195e49bcd10625cf4e46R174

Reply:
ipt[] is now guaranteed to be v4, v6.  Are clusterCidr[] and nodeIP?  It seems they are, though same comments as above - perhaps turning it into maps would be better